.github/workflows/test.yml

```diff
@@ -40,7 +40,10 @@ jobs:
       - uses: actions/setup-node@v2-beta
         with:
           node-version: '12'
-      - run: bash src/.buildkite/test/run.sh
+      # MySQL needs to be started manually https://github.blog/changelog/2020-02-21-github-actions-breaking-change-ubuntu-virtual-environments-will-no-longer-start-the-mysql-service-automatically/
```

Comment:
you can also use a standalone container for mysql instead of the one running on the VM

Reply:
What do you mean? like a cloud instance?